### PR TITLE
More inclusive comments - combine #220 and #221

### PR DESCRIPTION
**Pull request by MarkEWaite**
## More inclusive comments - Combine pull requests #220 and #221

- Make a comment more inclusive
- Make a comment more inclusive

## Checklist

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- ~~Ensure you have provided tests - that demonstrates feature works or fixes the issue~~

### CODE REVIEW

File: src/test/java/org/jenkinsci/plugins/workflow/EnvWorkflowTest.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public class EnvWorkflowTest {
     @Rule public JenkinsRule r = new JenkinsRule();
 
     /**
-     * Verifies if NODE_NAME environment variable is available on a slave node and on master.
+     * Verifies if NODE_NAME environment variable is available on an agent node and on the built-in node.
```
Comment (car-roll) [inline src/test/java/org/jenkinsci/plugins/workflow/EnvWorkflowTest.java:45]: I know the PR same inclusive comments, but it feels like we should update the test output as well?  i.e. `echo \"My name on built-in node...'

Comment (MarkEWaite) [reply src/test/java/org/jenkinsci/plugins/workflow/EnvWorkflowTest.java:45]: Pull request extended to include text changes in the test messages.

File: src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java [hunk 2]
```diff
@@ -424,13 +424,13 @@ private static void assertWorkspaceLocked(Computer computer, String workspacePat
     @Test public void acquireWorkspace() throws Throwable {
         sessions.then(r -> {
                 String slaveRoot = tmp.newFolder().getPath();
-                DumbSlave s = new DumbSlave("slave", slaveRoot, r.createComputerLauncher(null));
+                DumbSlave s = new DumbSlave("agent-007", slaveRoot, r.createComputerLauncher(null));
```
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java:427]: Why not just
```suggestion
                DumbSlave s = new DumbSlave("agent", slaveRoot, r.createComputerLauncher(null));
```
?

File: src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java [hunk 3]
```diff
@@ -424,13 +424,13 @@ private static void assertWorkspaceLocked(Computer computer, String workspacePat
...
                 s.setNumExecutors(2);
                 s.setRetentionStrategy(RetentionStrategy.NOOP);
                 r.jenkins.addNode(s);
                 WorkflowJob p = r.createProject(WorkflowJob.class, "demo");
                 p.setDefinition(new CpsFlowDefinition(
-                        "node('slave') {\n" + // this locks the WS
+                        "node('agent-007') {\n" + // this locks the WS
```
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java:433]:
```suggestion
                        "node('agent') {\n" + // this locks the WS
```
again

**Review by jglick — COMMENTED** [review]
Leaving this pending merge of #180 / #231 in case there are conflicts.

**Review by jglick — APPROVED** [review]

**Pull request merged**